src/Controller/MovieController.php

```diff
@@ -49,32 +58,82 @@ public function homePage(
         );
     }
 
-    #[Route('/movies/{id}', methods: ['GET'], name: 'movie_details_page')]
+    #[Route('/movies/{id}', methods: ['GET', 'POST'], name: 'movie_details_page')]
     public function movieDetailsPage(
-        string $id,
+        Movie $movie,
+        Request $request,
         #[MapQueryParameter] string $page = '1',
     ) {
-        if (!Uuid::isValid($id)) {
-            throw new BadRequestException("Invalid movie id");
-        }
+        $userReview = null;
+        $form = null;
+
+        if ($this->isGranted('IS_AUTHENTICATED_REMEMBERED')) {
```

Comment:
With this if statement, what if I login without ticking the checkbox "remember me"?